views/layouts/emails.njk

```diff
@@ -5,15 +5,12 @@
         <style>
             @import url('https://use.typekit.net/yyc1dgh.css');
         </style>
-        {% if appData.isDev %}
-            <link rel="stylesheet" type="text/css" href="{{ 'stylesheets/emails.css' | getCachebustedPath }}">
-        {% else %}
-            <link rel="stylesheet" type="text/css" href="/build/latest/stylesheets/emails.css">
-        {% endif %}
+        <link rel="stylesheet" type="text/css" href="/build/latest/stylesheets/emails.css">
     </head>
     <body>
         <div class="logo">
-            <img src="{% if appData.isDev %}/assets{% endif %}/images/site-logo.svg" alt="" />
+            {# We hard-code this path to avoid having to pass functions in for assembling a local path, and to prevent HTML parsers from inlining this image as a data URI #}
+            <img src="/images/logos/email.gif" alt="The National Lottery Community Fund logo" />
```

Comment:
This caused me lots of debugging pain. The [changes I introduced last week](https://github.com/biglotteryfund/blf-alpha/commit/003d8fc8a860ac85c0cb1ef7c8275693c8553eb0#diff-a8ac64883753b8f2e84be36c6cdf979eR16) broke this, d'oh. Basically, `juice`, which inlines our CSS for emails, does some magic to rewrite paths (so the CSS change wasn't needed), but I discovered that when it inlines SVGs, this doesn't work on GMail... or apparently [most other providers](https://css-tricks.com/a-guide-on-svg-support-in-email/#svg-email-inline). I opted instead for a gif which should always work.

Reply:
👍 Makes sense and good catch. Should/could this be a JPG instead of a GIF?